proto/protocolapis/astria/protocol/transactions/v1alpha1/action.proto

```diff
@@ -244,7 +218,7 @@ message BridgeSudoChangeAction {
   string fee_asset = 4;
 }
 
-message FeeChangeAction {
+message FeeChange {
```

Comment:
Thinking on the semantics of the fee change given the need for the rust type change from this. Would it make more sense to have this message be something like:

```proto
enum FeeVariable {
  UNKNOWN_FEE_VARIABLE = 0;
  TRANSFER_BASE_FEE = 1;
  ...
}

message FeeChange {
  FeeVariable fee_variable = 1;
  astria.primitive.v1.Uint128 value = 2;
}
```

Then the rust fee variables could be related to the enum and the `FeeChangeKind` might be able to go away? Probably not for this PR though.

Reply:
Discussed offline: generally agree. This action is not nice to read and its semantics are off. But will push that to a different PR.